CMakeLists.txt

```diff
@@ -235,8 +235,17 @@ endif()
 # Shared compiler flags used by all builds (debug, profile, release)
 # Allow COMPILER_FLAGS to be used as options from the ./configure without
 # forcing all the other options I want to add to be lost
-set(C_REAL_COMPILER_FLAGS "-Wall -pthread -g ${COMPILER_FLAGS}" CACHE STRING "common compiler options")
-set(CPP_REAL_COMPILER_FLAGS "-Wall -pthread -g ${CPP11_FLAGS} ${COMPILER_FLAGS}" CACHE STRING "common compiler options")
+set(C_REAL_COMPILER_FLAGS "-Wall -g ${COMPILER_FLAGS}" CACHE STRING "common compiler options")
```

Comment:
Why does this change require compiler flag changes?

Reply:
This fixes the Xcode 9.4 build break (with `-Werror` now added) - prior to Xcode 10, clang will warn about an unused parameter on `-pthread` at link time, whereas we need it at compile time. On gcc, we need it at both compile and link time.